CrazyCanvas/Include/States/ServerState.h

```diff
@@ -27,11 +29,15 @@ class ServerState :
 	void Tick(LambdaEngine::Timestamp delta) override final;
 	void FixedTick(LambdaEngine::Timestamp delta) override final;
 
+	bool OnPacketReceived(const LambdaEngine::PacketReceivedEvent& event);
+
 	bool OnKeyPressed(const LambdaEngine::KeyPressedEvent& event);
 
 	bool OnServerDiscoveryPreTransmit(const LambdaEngine::ServerDiscoveryPreTransmitEvent& event);
 
 private:
+	int32 m_ServerHostID = -1;
+	int32 m_ClientHostID = -1;
```

Comment:
This may be a better place than in the ClientSystem. Maybe you forgot to remove them in the ClientSystem